src/nvim/buffer_defs.h

```diff
@@ -535,7 +535,7 @@ struct file_buffer {
    * Character table, only used in charset.c for 'iskeyword'
    * 32 bytes of 8 bits: 1 bit per character 0-255.
    */
-  char_u b_chartab[32];
+  uint64_t b_chartab[32];
```

Comment:
`[4]` should be enough. Maybe the comment can read "bitset with 4*64=256 bits: 1 bit per character 0-255."